core/transaction.go

```diff
@@ -93,12 +93,12 @@ type TransactionReceipt struct {
 }
 
 type Transaction interface {
-	hash() *felt.Felt
+	Hash() *felt.Felt
```

Comment:
I would've prefered `GetHash`. changing the field name to `TransactionHash` sounds redundant, because it is already a Transaction object

Reply:
Go discourages `Get` Prefixes. That is why I don't like to use it until absolutely necessary. But I am happy to change the field to `Hash` and rename this function to `GetHash`.

@jelilat named it `TransactionHash`, I kept it because the thought the interface function `Hash()` would be used more so people wouldn't notice the stutter (that's what go calls `transaction.TransactionHash`).

Reply:
Got it, I wasnt suggesting that we should change it. Merely pointing it out

Reply:
Ok i am not going to change it for now